### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Farmeroz** [danielrab/gurps-easy-combat#1]
We tried this last weekend and it was very interesting - like what you have done so far!

One thing I noted as GM, running a combat with about a dozen NPCs, was that having it constantly pop up was an issue.  For a lot of the mooks I didn't need it.  I don't know if I'm missing something, but it would be nice to be able to choose by token whether or not it pops up, or maybe a pop up list of all tokens and then choose (mostly for a GM to use, of course).

Also, wondering about integration to call up the existing modifier box available through GGA, which is excellent?

Also, perhaps some "favourites" where deeper menu items could be player selected to appear up front on a favourites list for fast access?

**Comment by danielrab** [comment]
thanks for feedback. I have a few ideas how to handle the problem of it being spammy for a lot of mooks.
I definitely plan to look into using the existing modifiers bucket in some way when I'll be adding the ability to add modifiers to attack/damage/defense.
Some sort of favorites also sounds like a good idea.

all of that will take time to implement though, and there are currently issues that are more pressing with the module, like the fact that you can only deal damage and defend once per attack, or the fact that you can't add any modifiers to the damage.

**Comment by Farmeroz** [comment]
Yep, totally understand there are other priorities first.  Just wanted to express our group's enthusiasm and things we're thinking about having used it so far, and will do so as we go forward.

A way to manually pop it up before your turn would also be nice, so players could be "ready" for their attack or just to look at options.

It's a great idea for a module and I think it will help a lot.
